downstream/modules/platform/proc-aap-controller-yaml-restore.adoc

```diff
@@ -0,0 +1,58 @@
+[id="aap-controller-yaml-restore"]
+
+= Using YAML to recover the {ControllerNameStart} deployment
+See the following procedure for how to restore a deployment of the {ControllerName} using YAML. 
+
+.Prerequisite
+The external database must be a PostgreSQL database that is the version supported by the current release of {PlatformNameShort}.
+
+[NOTE]
+====
+{PlatformNameShort} {PlatformVers} supports PostgreSQL 13.
```

Comment:
Just an FYI that AAP 2.5 minimum requirement will be PostgreSQL 15. 
It might be a good idea to tie this number to an attribute. I can take a look at that :)